coral/src/app/features/topics/acl-request/TopicAclRequest.tsx

```diff
@@ -112,18 +51,21 @@ const TopicAclRequest = () => {
     },
   });
 
-  const { data: topicNames } = useQuery<TopicNames, Error>(["topic-names"], {
-    queryFn: () => getTopicNames({ onlyMyTeamTopics: false }),
-    keepPreviousData: true,
-    onSuccess: (data) => {
-      if (data?.includes(topicName)) {
-        return;
-      }
-      // Navigate back to Topics when topicName does not exist in the topics list
-      navigate("/topics");
-    },
-    enabled: topicName !== "",
-  });
+  const { data: topicNames } = useQuery<TopicNames, Error>(
+    ["topic-names", topicType],
+    {
+      queryFn: () => getTopicNames({ onlyMyTeamTopics: false }),
+      keepPreviousData: true,
+      onSuccess: (data) => {
+        if (data?.includes(topicName)) {
```

Comment:
Sure! This is a very temporary thing though, as the implementation of #483 will need to change it (otherwise we would navigate back if someone selects a topic, then change the environment to one that does not have the selected topic haha)

Reply:
💡  ok! 

... I'll update that in my own code then 😬